docs/advanced-usage/compute-backends/kubernetes/index.md

```diff
@@ -108,3 +100,87 @@ You can set `kubernetes_memory_limit` in every `rule` under `resources`.
         "docker://python:2.7-slim"
     ...
 ```
+
+## Custom job timeouts
+
+When a job exceeds the specified time limit, it will be terminated by Kubernetes and marked as failed.
+
+To set the job timeout, you can declare `kubernetes_job_timeout` in the specification of **each workflow step**.
+Time is measured in **seconds**.
+
+Read more about the job run time deadline limits in the [Kubernetes official documentation](https://kubernetes.io/docs/concepts/workloads/controllers/job/#job-termination-and-cleanup).
+
+You can configure the `steps` in the respective specifications for Serial, Yadage, CWL and Snakemake workflow engines.
+
+For **Serial**, you can set `kubernetes_job_timeout` in every `step` of workflow specification:
+
+```yaml hl_lines="6"
+   ...
+   steps:
+      - name: reana_demo_helloworld_job_timeout
```

Comment:
I think there's too much indentation here, 2 spaces should do.

Reply:
I will also fix the k8s memory limit. It has sometimes 3 spaces.